Add more private servers to default.json

i-yitsure opened the pull request:
Only for fun i added more of the top popular osu private servers. i can add more if the dev allow to me contribute, i will remove in case of the projects will shut down. He will notify by my contact e-mail. Thanks for listen to me

i-yitsure committed:
Add more private servers to default.json

Only for fun i added more of the top popular osu private servers. i can add more if the dev allow to me contribute, i will remove in case of the projects will shut down. He will notify by my contact e-mail. Thanks for listen to me

Airkek commented:
Seems ok
Thank you for contributing 
Please, add those servers into Models/Config.cs file

Airkek commented:
?

i-yitsure commented:
Okay, then i will do that when i will have time. Don't worry, i always will stay tuned with the topg forum. I mean i considerate im aproved for modify these archives ^^'

i-yitsure committed:
Actualizar README.md

Because i saw this commented by the original author to do (because i didn't get it how do the change and now im able for add more servers for the future. The description in the botton of the README.md sure will be removed in the future

Airkek commented:
most of those servers are dead for now hehe

i-yitsure commented:
> most of those servers are dead for now hehe

Recently I'm trying to revive my Github account for the Gist things, obviously I would be finding out which servers are not working. If anything, I'll prepare a separate list.
